app/services/getstream_webhook_service.rb

```diff
@@ -0,0 +1,97 @@
+# Parse getstream.io webhook reqest
+class GetstreamWebhookService
+  attr_reader :activity, :feed_id, :actor_id
+
+  def initialize(feed)
+    # TODO: See if multiple activities could actually happen
+    @activity = feed['new'].first
```

Comment:
This should almost definitely handle multiple activities.  If they're giving an array, it's clear that they expected that.